pkg/mistconnector/main.go

```diff
@@ -25,16 +24,12 @@ type MistConfig struct {
 	Version      string                  `json:"version,omitempty"`
 }
 
-func isIntType(value string) bool {
-	if _, err := strconv.Atoi(value); err == nil {
-		return true
-	}
-	return false
-}
-
 func isBoolType(value string) bool {
-	if _, err := strconv.ParseBool(value); err == nil {
-		return true
+	boolValues := []string{"true", "TRUE", "True", "false", "FALSE", "False"}
```

Comment:
or just `value.lowercase() in {"true", "false"}` (or its go equivalent)

Reply:
Updated